llvm/tools/llvm-readtapi/llvm-readtapi.cpp

```diff
@@ -138,8 +155,22 @@ int main(int Argc, char **Argv) {
     }
   }
 
+  if (Args.hasArg(OPT_compact))
```

Comment:
nit: this could be condensed into a single line. `Ctx.Compact = Args.hasArg(OPT_compact);`